Samples/ExtendedControlAndMetadata/EyeGazeAndBackgroundSegmentation/CameraKsPropertyHelper/PropertyInquiry.cpp

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#include "pch.h"
+#include "PropertyInquiry.h"
+#include "PropertyInquiry.g.cpp"
+#include "KSHelper.h"
+#include <winrt/Windows.Media.Devices.h>
+#include "BasicExtendedPropertyPayload.h"
+#include "BackgroundSegmentationPropertyPayload.h"
+#include "BackgroundSegmentationMaskMetadata.h"
+
+
+namespace winrt::CameraKsPropertyHelper::implementation
+{
+    /// <summary>
+    /// Get the data payload as an IPropertyValue associated with a Get command on an extended camera control
+    /// </summary>
+    /// <param name="controller"></param>
+    /// <param name="controlId"></param>
+    /// <returns></returns>
```

Comment:
>/// <returns></returns> [](http://example.com/codeflow?start=4&length=23)

Are the returns decorations in these comments supposed to be filled out?

Reply:
optional, VS does not populate them for you and it is a chore to update since you can't just use '///' atop your methof